/* 
 * File:   QEnvManager.h
 * Author: b4n92uid
 *
 * Created on 4 d√©cembre 2010, 00:04
 */

#ifndef QENVMANAGER_H
#define	QENVMANAGER_H

#include "ui_interface.h"

class QEnvManager
{
public:
    QEnvManager(Ui_mainWindow* uinterface);
    virtual ~QEnvManager();
private:

};

#endif	/* QENVMANAGER_H */

